exercise_050_cluster_cluster_singleton_akka_bootstrap/src/main/resources/application.conf

```diff
@@ -0,0 +1,97 @@
+akka {
+
+  management.cluster.bootstrap.contact-point-discovery {
+    service-name = "local-cluster"
+    discovery-method = config
+    required-contact-point-nr = 2
```

Comment:
Since there are 5 nodes total, it might make sense to make this at least 3?

Reply:
Will do.